Diagnostics/Get-NetView.PS1

```diff
@@ -1668,23 +1661,27 @@ function Completion {
         [parameter(Mandatory=$true)] [String] $OutZip
     )
 
-    # Calculate sizes
-    $dirs     = (Get-ChildItem $src -recurse | Measure-Object -property length -sum)
-    $dirssize = "{0:N2}" -f ($dirs.sum / 1MB) + " MB"
-
-    # Display file save location
-
-    Write-Host "`n`n"
-    Write-Host "Diagnostics Data:"
-    Write-Host "-----------------"
-    Write-Host $MyInvocation.PSCommandPath
-    Write-Host "Version: $version"
-    Write-Host "SHA256:  $((Get-FileHash -Path $($MyInvocation.PSCommandPath) -Algorithm ""SHA256"").Hash)`n"
+    # Zip output folder
     CreateZip -Src $Src -Out $OutZip
-    Write-Host "`n$Src"
-    Write-Host "Size:   $dirssize"
-    Write-Host "Dirs:  "(Get-ChildItem $Src -Directory -Recurse | Measure-Object | %{$_.Count})
-    Write-Host "Files: "(Get-ChildItem $Src -File -Recurse | Measure-Object | %{$_.Count})
+
+    $dirs = (Get-ChildItem $Src -Recurse | Measure-Object -Property length -Sum) # out folder size
+    $hash = (Get-FileHash -Path $MyInvocation.PSCommandPath -Algorithm "SHA256").Hash # script hash
+
+    # Display version and file save location
+    Write-Host "`n",
```

Comment:
Please preserve the old format.  Where individual items are separate Write-Host commands.  The original implementation was done for editing flexibility rather than runtime flexibility.  For example, one can apply modifiers to individual lines, whereas the new implementation is all or none.